glean-core/ios/GleanTests/Net/HttpPingUploaderTests.swift

```diff
@@ -15,10 +14,11 @@ class HttpPingUploaderTests: XCTestCase {
     override func tearDown() {
         // Reset expectations
         expectation = nil
-        OHHTTPStubs.removeAllStubs()
+        tearDownStubs()
     }
 
     func testHTTPStatusCode() {
+        Dispatchers.shared.setTestingMode(enabled: true)
```

Comment:
Maybe a comment that this is necessary for the uploader to not run-in-background?

Reply:
👍